tests/ui/consts/issue-17718-const-bad-values.rs

```diff
@@ -5,6 +5,7 @@ static mut S: usize = 3;
 const C2: &'static mut usize = unsafe { &mut S };
 //~^ ERROR: referencing statics in constants
 //~| ERROR: referencing statics in constants
+//~| ERROR: mutable references are not allowed
```

Comment:
There's a new diagnostic showing up here since the previously this triggered a Secondary const-op, now it triggers a Primary.